libs/wire-api/src/Wire/API/Event/FeatureConfig.hs

```diff
@@ -42,30 +42,31 @@ data Event = Event
   deriving (Eq, Show, Generic)
   deriving (A.ToJSON, A.FromJSON) via Schema Event
 
+arbitraryFeature :: forall cfg. (IsFeatureConfig cfg, ToSchema cfg, Arbitrary cfg) => Gen A.Value
+arbitraryFeature = toJSON <$> arbitrary @(LockableFeature cfg)
+
+class AllArbitraryFeatures cfgs where
+  allArbitraryFeatures :: [Gen A.Value]
+
+instance AllArbitraryFeatures '[] where
+  allArbitraryFeatures = []
+
+instance
+  ( IsFeatureConfig cfg,
+    ToSchema cfg,
+    Arbitrary cfg,
+    AllArbitraryFeatures cfgs
+  ) =>
+  AllArbitraryFeatures (cfg : cfgs)
+  where
+  allArbitraryFeatures = arbitraryFeature @cfg : allArbitraryFeatures @cfgs
```

Comment:
I think this might be possible with recursion on `SList` which reifies the type level list. You also need a csontraint on all elements, so perhaps smth like `cpara_SList` might work this time.